captum/attr/_utils/common.py

```diff
@@ -13,6 +13,14 @@ class ExpansionTypes(Enum):
     repeat_interleave = 2
 
 
+def safe_div(denom, quotient, default_value=None):
+    r"""
+        A simple utility function to perform `denom / quotient`
+        if the statement is undefined => result will be `default_value`
```

Comment:
After merging #131 this changes shouldn't show up here any more ...
You might need to do a rebase ...